features/release/barebone_tests.feature

```diff
@@ -85,10 +85,11 @@ Feature: Barebone tests
     And the event "unhandled" is true
     And the event "usage.callbacks" is not null
     And the event "usage.config" is not null
-    And the event "usage.config.staticallyLinked" equals the platform-dependent boolean:
-      | ios     | true  |
-      | macos   | @null |
-      | watchos | @null |
+    # TODO
+#    And the event "usage.config.staticallyLinked" equals the platform-dependent boolean:
+#      | ios     | true  |
+#      | macos   | @null |
+#      | watchos | @null |
```

Comment:
Is this still a TODO for this PR, or for a later PR?

Reply:
Have moved this to another feature file now, which isn't being run against the xcFramework fixture.